example/android/app/build.gradle

```diff
@@ -76,12 +76,13 @@ import com.android.build.OutputFile
  */
 
 project.ext.react = [
+    cliPath: "../../../node_modules/react-native/local-cli/cli.js",
     entryFile: "example/index.tsx",
     root: "../../../",
     enableHermes: false,  // clean and rebuild if changing
 ]
 
-apply from: "../../node_modules/react-native/react.gradle"
+apply from: "../../../node_modules/react-native/react.gradle"
```

Comment:
>Can you explain the addition of cliPath and the extra parent directory (../../node_modules/react-native to ../../../node_modules/react-native) changes?

In example you don't have `package.json` -> you have `package.json` only in root repo -> `node_modules` should be appear in root repo after `npm install`

It locks like someone install `node_modules` from root repo and just copy they to example directory:D 

>is cliPath a requirement now, and does react-native not exist in the example project's node_modules? I'm not sure why react-native is referenced from the module now instead of example app basically now, vs not referenced that way before, so I'm not sure this is needed or correct

When i build example app i was gets error something like "cliPath not fount". I was looks into another [project](https://github.com/react-native-webview/react-native-webview/blob/master/example/android/app/build.gradle#L79) and just do the same:)

Reply:
Interesting. I can see how this would work, and the react-native-webview example is a good reference. I don't like in general that this example is hand-crafted, I much prefer automatically generated examples (like https://github.com/thebergamo/react-native-fbsdk-next/blob/master/refresh-example.sh) so that none of this janitor work on examples is ever necessary, but since the example is currently hand-crafted, if this is what it takes to work, works for me